meta/recipes-kernel/linux/kernel-devsrc.inc

```diff
@@ -1,334 +0,0 @@
-	    cp -a --parents arch/arm/include/asm/opcodes.h $kerneldir/build/
-
-            cp -a --parents arch/arm64/kernel/vdso/*gettimeofday.* $kerneldir/build/
-            cp -a --parents arch/arm64/kernel/vdso/sigreturn.S $kerneldir/build/
-            cp -a --parents arch/arm64/kernel/vdso/note.S $kerneldir/build/
-            cp -a --parents arch/arm64/kernel/vdso/gen_vdso_offsets.sh $kerneldir/build/
-
-            cp -a --parents arch/arm64/kernel/module.lds $kerneldir/build/ 2>/dev/null || :
-
-            # 5.13+ needs these tools
-            cp -a --parents arch/arm64/tools/gen-cpucaps.awk $kerneldir/build/ 2>/dev/null || :
-            cp -a --parents arch/arm64/tools/cpucaps $kerneldir/build/ 2>/dev/null || :
-
-            if [ -e $kerneldir/build/arch/arm64/tools/gen-cpucaps.awk ]; then
-                 sed -i -e "s,#!.*awk.*,#!${USRBINPATH}/env awk," $kerneldir/build/arch/arm64/tools/gen-cpucaps.awk
-            fi
-	fi
-
-	if [ "${ARCH}" = "powerpc" ]; then
-	    # 5.0 needs these files, but don't error if they aren't present in the source
-	    cp -a --parents arch/${ARCH}/kernel/syscalls/syscall.tbl $kerneldir/build/ 2>/dev/null || :
-	    cp -a --parents arch/${ARCH}/kernel/syscalls/syscalltbl.sh $kerneldir/build/ 2>/dev/null || :
-	    cp -a --parents arch/${ARCH}/kernel/syscalls/syscallhdr.sh $kerneldir/build/ 2>/dev/null || :
-	    cp -a --parents arch/${ARCH}/kernel/vdso32/* $kerneldir/build/ 2>/dev/null || :
-	fi
-
-	# include the machine specific headers for ARM variants, if available.
-	if [ "${ARCH}" = "arm" ]; then
-	    cp -a --parents arch/${ARCH}/mach-*/include $kerneldir/build/
-
-	    # include a few files for 'make prepare'
-	    cp -a --parents arch/arm/tools/gen-mach-types $kerneldir/build/
-	    cp -a --parents arch/arm/tools/mach-types $kerneldir/build/
-
-	    # ARM syscall table tools only exist for kernels v4.10 or later
-            SYSCALL_TOOLS=$(find arch/arm/tools -name "syscall*")
-            if [ -n "$SYSCALL_TOOLS" ] ; then
-	        cp -a --parents $SYSCALL_TOOLS $kerneldir/build/
-            fi
-
-            cp -a --parents arch/arm/kernel/module.lds $kerneldir/build/ 2>/dev/null || :
-	fi
-
-	if [ -d arch/${ARCH}/include ]; then
-	    cp -a --parents arch/${ARCH}/include $kerneldir/build/
-	fi
-
-	cp -a include $kerneldir/build
-
-	cp -a --parents lib/vdso/* $kerneldir/build/ 2>/dev/null || :
-
-	cp -a --parents tools/include/tools/le_byteshift.h $kerneldir/build/
-	cp -a --parents tools/include/tools/be_byteshift.h $kerneldir/build/
-
-	# required for generate missing syscalls prepare phase
-	cp -a --parents $(find arch/x86 -type f -name "syscall_32.tbl") $kerneldir/build
-	cp -a --parents $(find arch/arm -type f -name "*.tbl") $kerneldir/build 2>/dev/null || :
-
-	if [ "${ARCH}" = "x86" ]; then
-	    # files for 'make prepare' to succeed with kernel-devel
-	    cp -a --parents $(find arch/x86 -type f -name "syscall_32.tbl") $kerneldir/build/ 2>/dev/null || :
```

Comment:
Would we get [these changes](https://github.com/ni/openembedded-core/commit/2796397355df8339f7d0b93de10de6b375b4f155) in kernel-devsrc.bb when we merge latest oe-core upstream?

Reply:
Yep. I've verified that. Those upstream changes will apply cleanly.